### ISSUE TITLE
Regression: It's not possible to create Cart Rule

### ISSUE THREAD

**Issue by getdatakick** [thirtybees/thirtybees#913]
It's not possible to create cart rule in 1.0.x/1.1.x branches due to javascript error:

```
admin.js?v=1.6.1.999:1393 Uncaught ReferenceError: priceDatabasePrecision is not defined
    at displayPriceValue (admin.js?v=1.6.1.999:1393)
```

Injected by commit bfa4116a78f55966412174d8a06736ed41889e68

**Comment by Traumflug** [comment]
Commit with a similar solution: bac6281bee8f1b23173b5ea103a44decb2bb8c8e

I've searched for a more central place for setting such constants already, but couldn't find a golden solution, yet.

**Comment by Traumflug** [comment]
Should be solved by 066055101dab44c3fc46d535752dd9cebcd55764
